### DISCUSSION TITLE
Better default settings

### DISCUSSION THREAD

**Discussion by Vnzq4Qje3zzU** [simple-login/app#822] [Feature Requests]
I think that can be used for general discussions about small improving to the default settings of new accounts, which mean a lot for people that are using an email alias service for the first time.

**Comment by Vnzq4Qje3zzU** [comment]
Making email replies more friendly:

1. Enable "Include sender address in reverse-alias" to make the reverse-alias more readable. The default setting is already leaking the sender address to the client (format "John Wick - john at wick.com").
2. Enable "Reverse Alias Replacement" when it's stable enough. I don't think most of the people will want to leak their true address in a simple reply.
3. Set "Sender Address Format" to just "John Wick". If I'm not mistaken, this should help with item 2 because there will be no transformation to the sender name, making the replacement easier and less error-prone.

**Comment by Vnzq4Qje3zzU** [comment]
Enable catch-all by default: It takes only a few minutes (or even seconds) between setting the DNS and starting to set the alias, but I'd rather receive some spam at admin@mydomain.tld than lose an important email in that period.

**Comment by nguyenkims** [comment]
@bx2KdaukwF thanks for the idea, making these default settings indeed make sense.
